.browserslistrc

```diff
@@ -0,0 +1,5 @@
+# Browsers that we support
```

Comment:
Actually, this file can be used by many different tools and is not specific to one of them. This is the idea of https://github.com/ai/browserslist. In sulu it's currently only used for css vendor prefixes. Should we add that to the comment?